src/server/migrations/migrateDatabase.js

```diff
@@ -0,0 +1,191 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const db = require('../models/database').db;
+const Migration = require('../models/Migration');
+const { compareSemanticVersion } = require('../util');
+const { log } = require('../log');
+
+// file needed to run database transaction
+const requiredFile = [];
+
+/**
+ * @param migrationItems extracted from registerMigration.js
+ * @returns {Array} of migration pair
+ */
+function printMigrationList(migrationItems) {
```

Comment:
`print` doesn't really make sense as a name for this method, since it doesn't actually print anything. You could also change the body to `return migrationItems.map(item => `${item.fromVersion} -> ${item.toVersion});`